content/posts/2015-09-04-what-is-technical-debt.md

```diff
@@ -81,7 +81,7 @@ within a software product over time:
 -   The predictability of results decreases, making management highly distrustful. (Producing accurate estimates for software with a high amount of debt is nearly impossible.)
 
 <div style="text-align:center">
-<img src="{{ site.baseurl }}/assets/blog/technical-debt/graphic-3.jpg" alt="A graph showing how the cost of change increases over time when you have technical debt, and that hurts your ability to meet user needs." width="400px">
+{% image "assets/blog/technical-debt/graphic-3.jpg" "A graph showing how the cost of change increases over time when you have technical debt, and that hurts your ability to meet user needs." %} width="400px">
```

Comment:
There's a stray `width` property after the end of the image tag, will probably render as text

Reply:
Nice catch, @beechnut --- fixed in https://github.com/18F/18f.gsa.gov/pull/3863/commits/ef47484d22fc375a653dac756676b07b69149b86#diff-07442e76559173210cb3172cbb544e5ef6b3747d7006186cd00730ae54ff7f23R84.

Reply:
By the way feel free to "resolve the conversation"

Reply:
Thanks, @cantsin! Done and done.